### ISSUE TITLE
Screen Reader: Preferences: The Zip code field is announced without a displayed note below it

### ISSUE THREAD

**Issue by elisenwazuoke** [Expensify/App#77378]
## Action Performed:
Precondition: The User should be logged in

1. Using Android TalkBack or iOS VoiceOver, open the Expensify app
2. Swipe to the 'Account' button at the bottom right corner of the screen and double-tap to activate
3. Swipe to the 'Profile' button and double-tap to activate
4. Swipe to the 'Address' button and double-tap to activate
5. Swipe to the 'Zip/Postcode' text field
6. Note how the field is announced.

## Expected Result:
The field should be announced together with the note below it, e.g.:
- 'Zip Code, e.g., 74213, 6978534'

## Actual Result:
The field is announced without a displayed note below it.

## Area issue was found in:
Preferences and  Subsections

## Failed WCAG checkpoints
WCAG 1.3.1 Info and Relationships
https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships

## User impact:
Users with different disabilities employ assistive technology to adapt content according to their specific needs. For example, users who are blind (using a screen reader) benefit when information conveyed through color is also available in text (including text alternatives for images that use color to convey information), and users who are deaf-blind and use braille (text) refreshable displays may be unable to access color-dependent information.

## Suggested resolution:
Include the note below the field in the accessibilityLabel or accessibilityHint to announce it when focusing the field.
https://reactnative.dev/docs/accessibility#accessibilityhint
https://reactnative.dev/docs/accessibility#accessibilitylabel

## Environment(s)
Samsung Galaxy S24 Ultra - Android 14, Partner - Native (No mobile browser)
Apple iPhone 11 - iOS 17.5.1, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos
![Image](https://github.com/user-attachments/assets/57d885fd-11f8-45fd-a4a0-addc44fa7db9)

https://github.com/user-attachments/assets/8f3d801b-01c1-4317-bb0e-5fdcd112e746

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7006354
**Issue reported by:** NA
**Slack conversation:** NA
